[PL-43116]: Doc updates for audit trail and audit stream done as part of doc review (#4275)
harness/developer-hub: docs/platform/governance/audit-trail/audit-trail.md

### Step: View an Audit Trail

> You can view Audit Trail data at Account/Org scope. This topic explains how to view Audit Trail data at the account scope.

Reviewer: ```suggestion
You can view Audit Trail data at the Account or Org scope. This topic explains how to view Audit Trail data at the account scope.
```

![](../../governance/audit-trail/static/audit-trail-01.png)

:::info IMPORTANT 

> By default, the pipeline execution audit events such as Pipeline Start, Pipeline End, Stage Start, and Stage End are not captured in Audit Trail. To enable capturing these events, you must enable the `Enable Pipeline Execution Audit Events` setting available under the pipeline category in the account level settings. This setting is only available at the account level. 

Reviewer: ```suggestion
By default, the pipeline execution audit events such as Pipeline Start, Pipeline End, Stage Start, and Stage End are not captured in the Audit Trail. To enable capturing these events, you must enable the `Enable Pipeline Execution Audit Events` setting available under the pipeline category in the account level settings. This setting is only available at the account level. 
```

   * Organization
   * Project
   * Resource Type
   * Resource Identifier
   * Action

   :::info note

>    The Resource Identifier operates in conjunction with the Resource Type. It allows you to employ the resource identifier for filtering audit events related to a specific resource using that identifier.

Reviewer: ```suggestion
   The Resource Identifier operates in conjunction with the Resource Type. It allows you to use the resource identifier to filter audit events related to a specific resource using that identifier.
```
